.project

```diff
@@ -26,4 +26,15 @@
 		<nature>org.eclipse.jdt.core.javanature</nature>
 		<nature>org.eclipse.wst.jsdt.core.jsNature</nature>
 	</natures>
+	<filteredResources>
```

Comment:
Are these changes related to this PR?

Reply:
Yes, this filters the test-bin directory from view in eclipse